ui/src/app/app.component.html

```diff
@@ -1,6 +1,6 @@
 <nav class="navbar navbar-expand-md navbar-dark bg-dark">
   <div class="container-fluid">
-    <a class="navbar-brand" href="#">MeTube</a>
+    <a class="navbar-brand" href="#">Youtube downloader</a>
```

Comment:
why this change?

Reply:
Ops sorry my bad, this can be deleted. I've used it in developer to see changes in UI and forget to delete